src/Orchestration/Adapter/DockerAPI.php

```diff
@@ -584,7 +584,7 @@ public function execute(
 
         $parsedResponse = json_decode($result['response'], true);
         if ($parsedResponse['Running'] === true || $parsedResponse['ExitCode'] !== 0) {
-            throw new Orchestration('Failed to execute command: '.$result['response'].' Exit Code: '.$parsedResponse['ExitCode']);
+            throw new Orchestration('Failed to execute command. Exit code: '.$parsedResponse['ExitCode']);
```

Comment:
The only reason this will be thrown is if the user provided build script does not completely successfully. Which is written to the $output reference, there's not really any more detail needed imo